src/components/App.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { useState } from 'react';
+import Page from './Page.jsx'
```

Comment:
```suggestion
import React from 'react'

import { useState } from 'react';

import Page from './Page.jsx'
```

의미상으로 구분되는 경우 빈 줄을 넣어 구분해주시면 좋아요.